### ISSUE TITLE
reproduce training

### ISSUE THREAD

**Issue by yemx21** [prgumd/SpikeMS#1]
Hi,
     I am very interested in your work.
     Can I have your suggestions on reproducing training stage?
     In your paper, I have a bit confused about your proposed spatio-temporal loss.

     Assume that I have an output spike tensor **e**=[batchsize, 2, 260, 346, timebins] and an masked target spike tensor **t**=[batchsize, 2, 260, 346, timebins]:

      (1) what does the spike project E(x)=sum_t (e(x)) mean? Is it equal to torch.sum(e, dim=4)? If it is, how can I implement binary cross-entropy for \lamada_bce (Eq.8)?

      (2) What is 1_f in Eq.9? Is it the mask of foreground spikes at t+delta_t or the mask labels at image domain?

      (3) How reduction is performed in the \lamada_bce and \lamada_spike (Eq.9)? Are they averaged over batchsize, height and width? 

   It will be great if you share your implementation of your proposed spatio-temporal loss.

Regards！
